db_stress_tool/db_stress_gflags.cc

```diff
@@ -450,6 +450,24 @@ DEFINE_int32(
     "[Integrated BlobDB] Enable writing blob files during flushes and "
     "compactions starting from the specified level.");
 
+DEFINE_bool(use_blob_cache, false, "[Integrated BlobDB] Enable blob cache.");
+
+DEFINE_bool(
+    use_shared_block_and_blob_cache, true,
+    "[Integrated BlobDB] Use a shared backing cache for both block "
+    "cache and blob cache. It only takes effect if use_blob_cache is enabled.");
+
+DEFINE_uint64(
+    blob_cache_size, FLAGS_cache_size,
+    "[Integrated BlobDB] Number of bytes to use as a cache of blobs. It only "
+    "takes effect if the block cache and blob cache are different "
+    "(use_shared_block_and_blob_cache = false).");
+
+DEFINE_int32(blob_cache_numshardbits, FLAGS_cache_numshardbits,
+             "Number of shards for the blob cache is 2 ** "
+             "blob_cache_numshardbits. Negative means use default settings. "
+             "This is applied only if blob_cache_size is non-negative.");
```

Comment:
It takes effect if `blob_cache_size` is (strictly) positive, right? Also, your comment about the block and blob caches having to be different also applies here.

Reply:
done